core/src/primitives/rectangle/mod.rs

```diff
@@ -321,21 +390,63 @@ impl Rectangle {
     /// );
     /// ```
     pub fn anchor_point(&self, anchor_point: AnchorPoint) -> Point {
+        Point::new(
+            self.anchor_x(anchor_point.x()),
+            self.anchor_y(anchor_point.y()),
+        )
+    }
+
+    /// Returns the X coordinate of an anchor edge.
```

Comment:
Maybe this would work better:

```suggestion
    /// Returns the X coordinate of a given anchor edge of the rectangle.
```

I found the original a bit too general, although it _is_ within the context of `Rectangle` so it doesn't matter that much.